tools/eslint-rules/set-proto-to-null-in-object.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+  meta: {
+    messages: {
+      error: 'Add `__proto__: null` to object',
+    },
+  },
+  create: function(context) {
+    return {
+      ObjectExpression(node) {
+        const properties = node.properties;
+        let hasProto = false;
+
+        for (const property of properties) {
+          if (property.key && property.key.name === '__proto__') {
```

Comment:
Can we verify that the value is `null`?

Reply:
yes, and I'm gonna make it fixable

Reply:
there are cases where we want to have `__proto__`  to not be `null`, see https://github.com/nodejs/node/blob/757c10414776b6f035ced453599527a841f47d6e/tools/eslint-rules/prefer-proto.js